packages/lesmis/src/api/api.js

```diff
@@ -38,6 +37,25 @@ const getApiUrl = () => {
 // withCredentials needs to be set for cookies to save @see https://developer.mozilla.org/en-US/docs/Web/API/XMLHttpRequest/withCredentials
 axios.defaults.withCredentials = true;
 
+const timeout = 45 * 1000; // 45 seconds
+
+const getRequestOptions = options => {
+  const locale = window.location.pathname.split('/')[1];
+  let requestOptions = { timeout, ...options };
+
+  if (locale !== DEFAULT_LOCALE) {
+    requestOptions = {
+      ...options,
+      params: {
+        ...options.params,
+        locale,
+      },
+    };
+  }
```

Comment:
Seems like we might as well send the locale through to the backend even if it's the default, so that
- the front end default isn't assuming the back end default is the same, so we could change them independently
- we drop a little bit of code and a `let` here
```suggestion
  return { timeout, ...options, params: { ...options.params, locale } };
```

There's a risk this causes the back end to run through "translate to English" unnecessarily, but I think the back end should be set up to avoid that if its default locale is passed through (cc @EMcQ-BES)